rust/private/rustc.bzl

```diff
@@ -737,7 +742,7 @@ def construct_arguments(
         force_link (bool, optional): Whether to add link flags to the command regardless of `emit`.
         stamp (bool, optional): Whether or not workspace status stamping is enabled. For more details see
             https://docs.bazel.build/versions/main/user-manual.html#flag--stamp
-        remap_path_prefix (str, optional): A value used to remap `${pwd}` to. If set to a falsey value, no prefix will be set.
+        remap_path_prefix (bool): Whether to apply the remap-path-prefix compiler setting.
```

Comment:
Could you preserve the `remap_path_prefix (str, optional)` and move the logic that picks it up from `_source_path_prefix` to clients; the implicit dependency on specific custom attributes in `ctx.attr` here is a thing we'd like to avoid at this layer, if possible. (We haven't been strictly enforcing this so you can see some instances of attributes used around).